client/account-recovery/reset-password/reset-password-confirm-form/index.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormTextInput from 'components/forms/form-text-input';
+import FormLabel from 'components/forms/form-label';
+import FormButton from 'components/forms/form-button';
+import { STRONG_PASSWORD } from 'lib/url/support';
+
+const ResetPasswordConfirmForm = ( props ) => {
+	const {
+		translate,
+		showPassword,
+	} = props;
+
+	return (
+		<Card>
+			<h2 className="reset-password-confirm-form__title">{ translate( 'Reset your password' ) }</h2>
+			<FormLabel className="reset-password-confirm-form__text-input-label" htmlFor="password">
+				{ translate( 'New password' ) }
+			</FormLabel>
+			<a className={ classNames( 'reset-password-confirm-form__show-hide-toggle', { show: showPassword } ) } href="#">
+				{ showPassword ? translate( 'Show' ) : translate( 'Hide' ) }
+			</a>
+			<FormTextInput className="reset-password-confirm-form__password-input-field" id="password" />
+			<FormButton className="reset-password-confirm-form__button generate-password-button" isPrimary={ false }>
+				{ translate( 'Generate strong password' ) }
+			</FormButton>
+			<p className="reset-password-confirm-form__description">
+				{ translate(
+					'{{a}}Great passwords{{/a}} use upper and lower case characters, numbers, and symbols like !/"$%&',
```

Comment:
Hi! I've found a possible matching string that has already been translated **44** times:
`translate( '<a href="%1$s" target="_blank">Great passwords</a> use upper and lower case characters, numbers, and symbols like <em>%2$s</em>.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).